AzureMonitorAgent/agent.py

```diff
@@ -302,7 +302,7 @@ def install():
     vm_dist, vm_ver = find_vm_distro('Install')
 
     # Check if SUSE 15 VMs have /sbin/insserv package (required for AMA 1.14.4+)
-    if (vm_dist.lower().startswith('suse') or vm_dist.lower().startswith('sles')) and vm_ver.startswith('15'):
+    if (vm_dist.lower().startswith('suse') or vm_dist.lower().startswith('sles') or vm_dist.lower().startswith('opensuse')) and vm_ver.startswith('15'):
```

Comment:
nit: no need to `.lower()` these. the function find_vm_distro will convert to lowercase before returning the values